### ISSUE TITLE
Mypal 68.12.5b xp86 fully patch updated crashes on startup.

### ISSUE THREAD

**Issue by vertaa** [Feodor2/Mypal68#136]
When loading the newer mypal68.12.5b verson it crashes on startup. I am running windows xpsp3x86.

Note: It does work in safe mode.

I have attached a crash debug log.

[WER6665.dir00.zip](https://github.com/Feodor2/Mypal68/files/9303518/WER6665.dir00.zip)

**Comment by Feodor2** [comment]
This is not a regular log which i request, i dont know how to handle it
Please provide regular winxp error log drwatson

**Comment by Feodor2** [comment]
So i opened it and have looked
There is no point of the crash though i see some files are suspicios, certainly the d2d1. What has put it into the winxp system is a question. Remove it from there and put only where other program may needed
![image](https://user-images.githubusercontent.com/19492771/184251187-ec36ea4a-da7c-4db7-8ac4-a1b9a02e64e0.png)

**Comment by aric2746** [comment]
Feodor2, My apologizes. 

trayit! allows you to minimize taskbar programs into the system tray.
mdnsNSP belongs to a program to setup WIFI prior to windows XP Wireless Zero Config (WZC).
d2d1.dll appears to be part of directx2??

So I'll disable the following three programs. It does work in safe mode. 

What are the settings for drwatson "drwtsn32.exe?", I will produce another log as instructed. All previous versions of mypal and mypal68 have worked perfectly on this system. The system is as basic as it can be, so hopefully I can get you the information you need to stop the crash.

**Comment by aric2746** [comment]
Removed all three programs and moving d2d1.dll out of the system32 directory allowed Mypal 68.12.5b to load. 

d2d1.dll maybe needed on the system by other programs. Can Mypal 68.12.5.b not pay attention to this dll so I can put it back just in case.

If for future use is there a doc/howto for your drwatson"drwtsn32.exe?" configuration so I can provide the information to you as you request.

**Comment by Feodor2** [comment]
d2d1 i think a reason of the crash, do you know what excatly needs it on the winxp. Other programs return you may try to.
Drwatsin log is inside All Users\Application Data\Microsoft\Dr Watson usually, so look there first if there is already exists the log please upload it here.

**Comment by aric2746** [comment]
![dr watson](https://user-images.githubusercontent.com/67488797/184297249-a7f36a91-edd3-40d9-9a1c-139f9126fe94.PNG)

This is an interesting test. I had mypal 68.12.5b loaded, then I copied the file d2d1.dll in c:\windows\system32. I expected mypal 68.12.5b to crash, instead it loaded fine. No log or dump file was created in All Users\Application Data\Microsoft\Dr Watson.

![drwtsn32](https://user-images.githubusercontent.com/67488797/184297420-57fe8767-8a3c-40d3-a35e-18a437ac41e9.PNG)

A log file was created but is empty and no dump in All Users\Application Data\Microsoft\Dr Watson.

![d2d1](https://user-images.githubusercontent.com/67488797/184297588-135f6582-9365-4b2e-8d70-410e35d75678.PNG)

Microsoft D2D Library 6.1.6519.1 I have attached the dll file.

[d2d1.zip](https://github.com/Feodor2/Mypal68/files/9314210/d2d1.zip)

**Comment by AroKol78** [comment]
I have winXP3sp32bit, gpu NVIDIA card,the latest d3d9 drivers, I don't have this file `c:\windows\system32\d2d1.dll`, games and programs for XP work.
> This dll appears in Directx 10 NYV 

I think XP doesn't have an API for Directx 10 and d2 (unless you hacked something)
`gfx.direct2d.disabled` `....azure.backends direct2d1.1,skia,cairo`
does it work on Vista?
you may need to reinstall the appropriate directX.

**Comment by aric2746** [comment]
AroKol78,

Yeah I had Directx 10 NYV for a long time. The crash never happened until the latests release, Mypal 68.12.5b.

I did uninstall dirextx 10, but noticed the "copy and paste" did not work correctly on discord anymore. So I just removed d2d1.dll, and everything started working correctly, nothing appears out of place.

I plan on another uninstall and this time I will use a program to restore directx9 in safe mode https://tnabbs.ax.lt/directx-dx9uninstaller.zip. I'll then run Windows Update and select all the directx9 updates.

I do not run vista anymore.
